app/models/upload.rb

```diff
@@ -230,9 +230,9 @@ def rebake_posts_on_old_scheme
     self.posts.where("cooked LIKE '%/_optimized/%'").find_each(&:rebake!)
   end
 
-  def update_secure_status
+  def update_secure_status(secure_override: nil)
```

Comment:
@eviltrout my intent must not be clear here. I've set it to nil because this parameter will either update the `secure` status to true or false depending on what is passed here. I could rename the variable to something like `secure_override_value` or change the parameter to an `options` parameter?

Reply:
I see, so the API will optionally update that attribute if provided as `true/false`, but if `nil` it will remain the same? 

I think that's OK then, sorry for the confusion.